### DISCUSSION TITLE
Change Password Enhancements

### DISCUSSION THREAD

**Discussion by carlosmbe** [specify/specify7#5836] [Feature Requests]
The Change Password experience could be slightly better.

Firstly, could we have the `fill in field` prompt on top rather than on the bottom. On chrome the auto fill ends up blocking it.
<img width="454" alt="Screenshot 2023-06-29 at 3 42 53 PM" src="https://github.com/specify/specify7/assets/53784701/54b6440a-2f6b-4349-a323-b716109e08e7">

Secondly, could we add a see password button so I can see what I typed in and make sure it's correct. 
![3e3065a30e5e586a1e65f53cc2e79d2b](https://github.com/specify/specify7/assets/53784701/92c2fb98-381f-4280-ab47-531065655ea7)

**Comment by maxpatiiuk** [comment]
> Firstly, could we have the fill in field prompt on top rather than on the bottom

This is controlled by the browser, not us.

> Secondly, could we add a see password button so I can see what I typed in and make sure it's correct

Some browsers provide this feature. Chrome apparently doesn't 

We could implement it ourself, but we have a repeat password field instead - that was simpler to implement (and thus less buggy)
